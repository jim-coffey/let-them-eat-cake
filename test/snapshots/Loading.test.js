import React from 'react';
import renderer from 'react-test-renderer';

import Loading from '../../src/components/Loading';

describe('Loading (Snapshot)', () => {
  it('renders correctly', () => {
    const loadingText = 'Now Loading';
    const speed = 250;

    const tree = renderer.create(
      <Loading text={loadingText} speed={speed} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
