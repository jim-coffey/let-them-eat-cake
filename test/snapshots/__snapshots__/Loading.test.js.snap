// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading (Snapshot) renders correctly 1`] = `
<div>
  <p
    style={
      Object {
        "fontSize": "35px",
        "textAlign": "center",
      }
    }
  >
    Now Loading
  </p>
</div>
`;
